ethereum-network-partition-test/eth2_starlark_package.star

```diff
@@ -0,0 +1,44 @@
+eth_package = import_module("github.com/kurtosis-tech/eth2-package/main.star")
+
+PARTICIPANT_CONFIG = {
+    "el_client_type": "geth",
+    "el_client_image": "ethereum/client-go:v1.10.25",
+    "cl_client_type": "lighthouse",
+    "cl_client_image": "sigp/lighthouse:v3.1.2",
+}
+
+# This is hardcoded in the eth2_package.
+# TODO: Maybe rework the object this package returned to get those node ids dynamically
```

Comment:
oh wonder if we need "get_services_within_enclave" as a Starlark method or something 😅